cli/src/main/java/org/apache/uniffle/cli/UniffleCLI.java

```diff
@@ -25,28 +25,53 @@
 
 import org.apache.uniffle.AbstractCustomCommandLine;
 import org.apache.uniffle.UniffleCliArgsException;
+import org.apache.uniffle.api.AdminRestApi;
+import org.apache.uniffle.client.UniffleRestClient;
+
 
 public class UniffleCLI extends AbstractCustomCommandLine {
 
   private static final Logger LOG = LoggerFactory.getLogger(UniffleCLI.class);
   private final Options allOptions;
   private final Option uniffleClientCli;
   private final Option uniffleAdminCli;
+  private final Option checkerClass;
+  private final Option coordServer;
+  private final Option coordPort;
+  private final Option refreshCheckerCli;
   private final Option help;
+  protected UniffleRestClient client;
 
   public UniffleCLI(String shortPrefix, String longPrefix) {
     allOptions = new Options();
     uniffleClientCli = new Option(shortPrefix + "c", longPrefix + "cli",
         true, "This is an client cli command that will print args.");
     uniffleAdminCli = new Option(shortPrefix + "a", longPrefix + "admin",
         true, "This is an admin command that will print args.");
+    refreshCheckerCli = new Option(shortPrefix + "rc", longPrefix + "refreshChecker",
+        false, "This is an admin command that will refresh access checker.");
+    checkerClass = new Option(shortPrefix + "cc", longPrefix + "checkerClass",
```

Comment:
The class name should be progagted to this options. It's weired and complex. But I have no ideas on this

Reply:
I changed the input for the class name to `-c`, which looks more comfortable than the previous `-cc`.

Reply:
I think we can remove the checkerClass parameter, as the operation to update the configuration should be idempotent and there is no need to consider updating just one specific checker.So, I remove the the checkerClass parameter.